testonly/integration/clientserver.go

```diff
@@ -39,9 +41,21 @@ var (
 	sleepBetweenRuns = 100 * time.Millisecond
 	timeSource       = util.SystemTimeSource{}
 	// PublicKey returns the public key that verifies responses from this server.
-	PublicKey, _ = keys.NewFromPublicPEM(to.DemoPublicKey)
```

Comment:
This seems like we're adding some complexity here, do you think we can simplify it?